Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -516,18 +516,13 @@ private extension ProductStore {
     func generateProductDescription(siteID: Int64,
                                     name: String,
                                     features: String,
-                                    languageCode: String,
                                     completion: @escaping (Result<String, Error>) -> Void) {
         let prompt = [
-            "Your task is to help a marketing team create a description for a retail website of a product based on the " +
-            "properties delimited by triple backticks at the end.",
+            "Write a description for a product with title \(name) and features: \(features).",
```

Comment:
I'd recommend marking the variables (name and features) in a quote or the single/double/triple backtick marks (less likely to cause issues when the variables also contain quotation marks), so that the model can easily distinguish them from the rest of the prompt. In Woo AI, [they seem to be using quotation marks](https://github.com/Automattic/woo-ai-production/blob/705a21ad39ca978cdd426a6923706d6bfe557764/src/product-description/product-description-button-container.tsx#L113).

Reply:
Updated in 2e1ba12bcf59b0e409f0a606fdb93ebdbdbf378b with backticks, quotation marks result in the response for me.